Source/Core/Common/IniFile.cpp

```diff
@@ -39,6 +39,12 @@ void IniFile::ParseLine(const std::string& line, std::string* keyOut, std::strin
 
 const std::string& IniFile::NULL_STRING = "";
 
+IniFile::Section::Section() = default;
+
+IniFile::Section::Section(std::string name_) : name{std::move(name_)}
```

Comment:
Maybe an *nitpicking*, but why the underscore in the parameter if you are going to use class param initializer